src/app/components/streetview-logs/streetview-logs.component.html

```diff
@@ -0,0 +1,93 @@
+<div>
+  <div *ngIf="progressNotifications == undefined || progressNotifications.length == 0; else showLogs">
+    No publish tasks are currently in progress!
+  </div>
+
+  <ng-template #showLogs>
+    <div *ngFor="let notification of progressNotifications">
+      <div class="upload-box">
+        <div class="upload-box-name"
+             style="max-width: 150px; white-space: nowrap; overflow: hidden; text-overflow: ellipsis"
+             [tooltip]="notification.uuid">
+          {{notification.uuid}}
+        </div>
+
+        <div class="upload-info-box" *ngIf="notification.status == 'error'">
+          <div class="upload-info-status">
+            <span class="upload-info-message">
+              Upload failed!
+            </span>
+            <span>
+              <i class="fas fa-exclamation-triangle" style="color: #DB3A42; cursor: pointer" (click)="openDetailLogModal(notification)"></i>
+            </span>
+          </div>
+          <div class="upload-info-button-container">
+            <button class="button tiny upload-info-button" (click)="deleteErrorLog(notification)">
+              Delete log
+            </button>
+          </div>
+        </div>
+
+        <div class="upload-info-box" *ngIf="notification.status == 'created'">
+          <div class="upload-info-status">
+            <span class="upload-info-message">
+              {{notification.message}}
+            </span>
+
+            <span>
+              <i class="fas fa-circle-notch fa-spin"></i>
+            </span>
+          </div>
+          <div class="upload-info-button-container">
+            <button class="button tiny upload-info-button" (click)="deleteDoneLog(notification)">
+              Delete log
+            </button>
+          </div>
+        </div>
+
+        <div class="upload-info-box" *ngIf="notification.status == 'success'">
+          <div class="upload-info-status">
+            <span class="upload-info-message">
+              Upload sucess!
```

Comment:
```suggestion
              Upload success!
```

or maybe checkmark instead of text?